torch/_functorch/aot_autograd.py

```diff
@@ -3851,6 +3855,13 @@ def fn_to_trace(*args):
             num_params_buffers=len(params_and_buffers_flat),
             no_tangents=True,
         )
+
+    # For debug purposes the user may want to inspect the joint graph.
+    # Skipping the flatten_joint stage perserves the node.meta information
+    # available in the joint fwd/bwd graph fx_g.
+    if skip_flatten_joint:
+        trace_joint = False
+
```

Comment:
I went ahead and removed all the changes I made to aot_export_module.  I made the changes initially because I needed a way to write a test case for the sequence_nr() related changes.  I found a workaround so I only have to modify the test case.